runtime/kusama/src/lib.rs

```diff
@@ -2098,6 +2098,12 @@ sp_api::impl_runtime_apis! {
 				hex_literal::hex!("06de3d8a54d27e44a9d5ce189618f22db4b49d95320d9021994c850f25b8e385").to_vec().into(),
 				// The transactional storage limit.
 				hex_literal::hex!("3a7472616e73616374696f6e5f6c6576656c3a").to_vec().into(),
+				// XcmPallet SupportedVersion
+				hex_literal::hex!("1405f2411d0af5a7ff397e7c9dc68d19908b4dae59e7d658f23d261c5e5db22b").to_vec().into(),
```

Comment:
Looking at the code; this is a map and probably not *entirely* read each block.So let's not whitelist that. (same for the other runtimes)  
```suggestion
```